cli_tools/gce_vm_image_import/main_test.go

```diff
@@ -15,10 +15,12 @@
 package main
 
 import (
+	"cloud.google.com/go/storage"
```

Comment:
this is also considered external and should go to the second group (even though it's google)